scout/adapter/client.py

```diff
@@ -8,7 +8,11 @@
 import logging
 
 from pymongo import MongoClient
-from pymongo.errors import ConnectionFailure, OperationFailure, ServerSelectionTimeoutError
+from pymongo.errors import (
```

Comment:
Hm, that's a lot of unrelated import formatting. Are you sure you have line length set for your editor isort?